custom_components/adjustable_bed/support_report.py

```
@@ -0,0 +1,238 @@
+            "Enable debug logging and reproduce the issue to capture logs.",
+        })
+
+    # Limit and return most recent
+    return logs[-MAX_LOG_ENTRIES:]
+
+
+def save_support_report(
+    hass: HomeAssistant, report: dict[str, Any], address: str
+) -> Path:
+    """Save support report to a JSON file in the config directory."""
+    timestamp = datetime.now(timezone.utc).strftime("%Y%m%d_%H%M%S")
```

Ruff (0.14.13)

Use `datetime.UTC` alias

Convert to `datetime.UTC` alias

(UP017)